baseservice/iodevices/file_system/file_system_device_manager_base.py

```diff
@@ -83,9 +85,3 @@ def connect(self):
             create_dir_if_not_exists(self._bookkeeping_folder)
         except Exception as e:
             raise KwargsException('Error creating directories') from e  # TODO: raise another type of exception
```

Comment:
Is there a need to wrap this exception? Why not let it raise by itself?

Reply:
I prefer that the exceptions raised are my own... (added params and shit...)

can you think of a down side?